app/models/concerns/solidus_multi_domain_enhancements/order_attributes.rb

```diff
@@ -21,6 +21,10 @@ def generate_order_number(options = {})
       end
     end
 
+    def locale
+      self[:locale].try(:to_sym) || store.try(:default_locale)
```

Comment:
The  safe navigation operator (`&.`) isn't compatible with the minimum version on Ruby that Solidus supports - which is 2.1 - would be awesome though 😞